src/app_service/service/community/service.nim

```diff
@@ -98,6 +102,16 @@ QtObject:
       error "error: ", errDesription
       return
 
+    self.events.on(SignalType.Message.event) do(e:Args):
```

Comment:
@jrainville I just commit a minor thing here instead of writing a comment, just removed `*` to make `doConnect` a private method. Hope that you don't mind.